src/CAREUI/icons/CustomIcons.tsx

```
@@ -169,3 +169,344 @@ export const HealthWorkerIcon: React.FC<IconProps> = ({
+
+export const BedAvailableSelected: React.FC<IconProps> = ({
+  className = "size-4",
+}) => (
+  <svg
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    className={className}
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)